std/array.d

```diff
@@ -1443,20 +1443,30 @@ if (isInputRange!S && !isDynamicArray!S)
 }
 
 /++
-Eagerly split the string $(D s) into an array of words, using whitespace as
-delimiter. Runs of whitespace are merged together (no empty words are produced).
+Eagerly splits `range` into an array, using `sep` as the delimiter.
 
-$(D @safe), $(D pure) and $(D CTFE)-able.
+When no delimiter is provided, strings are split into an array of words,
+using whitespace as delimiter.
+Runs of whitespace are merged together (no empty words are produced).
+
+The `range` must be a $(REF_ALTTEXT forward _range, isForwardRange, std,_range,primitives).
+The separator can be a value of the same type as the elements in `range`
+or it can be another forward `range`.
+
+Note:
```

Comment:
this is already covered in the see also section

Reply:
This was explicitly mentioned in this bug report, but fair enough ...